bin/docker_run_default

```diff
@@ -31,4 +31,4 @@ docker run \
   -e COOKIE_SIGNATURE=$COOKIE_SIGNATURE \
   -e PORT=$PORT \
   -e MONGO_URL=$MONGO_URL \
```

Comment:
This is not required anymore, and probs we need to replace this with PostgreSQL variables